app/controllers/api/documents_controller.rb

```diff
@@ -26,9 +27,10 @@ def documents
       @documents ||= DocumentProvider.new(
         params.permit(
           :keyword,
+          :order_by_date,
           :tag, tag: [],
-          document_type: [],
-          blocks: [:identifier, :value, value: []]
+                document_type: [],
+                blocks: [:identifier, :value, value: []]
```

Comment:
@margOnline rubocop aligned this two lines to the key with an array as value from the previous line, and it makes sense if you look at it more closely:

```
:string,
:string,
:string, array: [values],
         array: [values],
         array: [values]
```

Reply:
I find this confusing because it appears as if `document_type` and `blocks` are parameters nested within `tags` and they are not. I wonder if rubocop is confused by setting on the same line `tag` as a String or  `tag` as an Array? Perhaps the following would be better?
```
:tag, 
tag: [],
document_type: [],
blocks: [:identifier, :value, value: []]

Reply:
@margOnline as you are saying, the problem is that we had:

```
:tag, tag: [],
```

which, I thought, was a clear decision to keep strings and arrays separated, so rubocop was actually clever to follow it in my opintion.

The indentation doesn't always indicate nested elements as we know, but I agree it can be confusing in this circumstance, so I have changed it. ⛱